lib/customer.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../lib/order'
+require 'csv'
+
+class Customer
+  @@customers = []
+
+  attr_reader :id
+  attr_accessor :email, :address
+  ID = 0
+  EMAIL = ""
+  ADDRESS = {}
```

Comment:
I think these constants `ID`, `EMAIL`, and `ADDRESS` end up not being used in this class!